src/vellum/workflows/events/tests/test_event.py

```diff
@@ -73,6 +74,7 @@ class MockWorkflow(BaseWorkflow[MockInputs, BaseState]):
                 "name": "workflow.execution.initiated",
                 "body": {
                     "workflow_definition": {
+                        "id": str(uuid4_from_hash(MockWorkflow.__qualname__)),
```

Comment:
[nit] slight pref for setting these two as hardcoded files in the top of the file and using in tests